ts/features/itwallet/common/store/reducers/index.ts

```diff
@@ -35,6 +35,19 @@ const persistConfig: PersistConfig = {
   version: CURRENT_REDUX_ITW_STORE_VERSION
 };
 
+const credentialsPersistConfig: PersistConfig = {
+  key: "itWalletCredentials",
+  storage: itwCreateCredentialsStorage(),
+  version: CURRENT_REDUX_ITW_CREDENTIALS_STORE_VERSION
+};
```

Comment:
My rationale here was to have all persist configurations in a centralized place.

Since this reducer is using a custom storage I can move the persist config in the credentials reducer, but I would prefer to leave it here for consistency (the other persist config is here as well).